base/libgit2/walker.jl

```diff
@@ -1,5 +1,31 @@
 # This file is a part of Julia. License is MIT: https://julialang.org/license
 
+"""
+    GitRevWalker(repo::GitRepo)
+
+A `GitRevWalker` *walks* through the *revisions* (i.e. commits) of
```

Comment:
Maybe something like
> A `GitRevWalker` is an iterable collection of the revisions (i.e. commits) of a git repository `repo`. Iteration over the collection walks the individual commits and returns `(oid, repo)` tuples, where `oid` is a [`GitHash`](@ ref) and `repo` is the given repository.
>
> Like many other such collections, `GitRevWalker` supports [`map`](@ ref) and [`count`](@ ref). The latter could, for example, be used to determine the number of commits in a repository made by a particular author.

Also TIL that `LibGit2.map` is a thing. Do we have a docstring that points to that specifically?